airflow/providers/airbyte/__init__.py

```diff
@@ -27,7 +27,7 @@
 
 __all__ = ["__version__"]
 
-__version__ = "3.6.0"
+__version__ = "3.6.1"
```

Comment:
You shouldn't touch this version, it is bumped by Release Manager

Reply:
Thanks - I've sorted that.